frontend/app/services/zuora/ZuoraActions.scala

```diff
@@ -352,3 +352,25 @@ case class UpgradePlan(subscriptionId: String, subscriptionRatePlanId: String, n
     </ns1:amend>
   }
 }
+
+case class CreateFreeEventUsage(accountId: String, description: String, quantity: Int, subscriptionId: Option[String]) extends ZuoraAction[CreateResult] {
```

Comment:
I think the `subscriptionId` probably shouldn't be an option - if we can't find a subscription for the user, something is very wrong!